src/GlobalScreen/Scope/MetaBar/Factory/NotificationCenter.php

```diff
@@ -2,6 +2,7 @@
 
 use ILIAS\GlobalScreen\Identification\IdentificationInterface;
 use ILIAS\GlobalScreen\Scope\MetaBar\Collector\Renderer\NotificationCenterRenderer;
+use ILIAS\GlobalScreen\Scope\Notification\Factory\isItem as notaIsItem;
```

Comment:
Why did you introduce this alias?

Reply:
To prevent mistakings with the isItem in the same directory. This is another item

Reply:
Then I suggest to call it `IsNotificationItem`